services/core-api/app/__init__.py

```diff
@@ -97,11 +99,94 @@ def register_routes(app):
     root_api_namespace.add_namespace(orgbook_api)
     root_api_namespace.add_namespace(EMLI_contacts_api)
 
-    # Healthcheck endpoint
+    # General Service status
     @root_api_namespace.route('/health')
     class Healthcheck(Resource):
+        def get_health(self):
```

Comment:
This isn't being used any more

Reply:
It will be used by external monitoring services - Uptime robot